### PR TITLE
css/

### PR DESCRIPTION
**Pull request by PacoGD**

### CODE REVIEW

File: staff/PacoGD/css-layout-simple-adaptive/styles/main.css [hunk 1]
```diff
@@ -0,0 +1,2 @@
+@import "desktop.css" screen and (min-width: 768px);
+@import "mobile.css" screen and (max-width: 768px);
```
Comment (EzequielBurgosL) [inline staff/PacoGD/css-layout-simple-adaptive/styles/main.css]: Some of the CSS properties can be listed in the "main.css" document to reduce your code size

File: staff/PacoGD/css-layout-simple-adaptive/styles/mobile.css [hunk 2]
```diff
@@ -0,0 +1,41 @@
+html,
```
Comment (manuelbarzi) [inline staff/PacoGD/css-layout-simple-adaptive/styles/mobile.css]: TIP mobile and desktop share some rules, may the be centralised in main.css.

Comment (PacoGD) [reply staff/PacoGD/css-layout-simple-adaptive/styles/mobile.css]: Okey!

File: staff/PacoGD/css-layout-simple-adaptive/styles/main.css (hunk 1 above)
Comment (PacoGD) [reply staff/PacoGD/css-layout-simple-adaptive/styles/main.css]: Okey

Comment (PacoGD) [reply staff/PacoGD/css-layout-simple-adaptive/styles/main.css]: done

File: staff/PacoGD/cv/styles/main.css [hunk 3]
```diff
@@ -0,0 +1,60 @@
+html{
+    background-color: aliceblue;
+}
+
+body {
+    /* background: rgb(150, 173, 111); */
```
Comment (agandia9) [inline staff/PacoGD/cv/styles/main.css:6]: delete comments before push!

File: staff/PacoGD/cv/styles/main.css [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    margin-left: 1.4%;
+}
+article{
+    border: 1px solid rgb(3, 0, 0);
+    box-sizing: border-box;
+    border-radius: 25px;
+    margin: 1%;
+}
+
+/* img {
```
Comment (agandia9) [inline staff/PacoGD/cv/styles/main.css:52]: !!

File: staff/PacoGD/js/challenges/to-camel-case/scripts/toCamelCase.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+function toCamelCase(str){
+    //TODO
+
+    return  // res -> "helloMyWorld"
```
Comment (agandia9) [inline staff/PacoGD/js/challenges/to-camel-case/scripts/toCamelCase.js]: ¿?

File: staff/PacoGD/portfolio/styles/main.css [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+    background-position: center;
+    background-size: cover;
+    border-radius: 25px;
+    background-image: url("https://i2.wp.com/hipertextual.com/files/2015/05/avengers_age_of_ultron_2015_movie-wide.jpg?resize=670%2C413&ssl=1");
+}
+
+.grid-item {
+    padding: 60px;
+    border: 1px solid black;
+    /* background: rgb(163, 180, 3); */
```
Comment (agandia9) [inline staff/PacoGD/portfolio/styles/main.css:32]: !!

File: staff/PacoGD/portfolio/styles/main.css [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+    grid-column: 5/7;
+    background-image: url("https://pics.filmaffinity.com/las_leyes_de_la_termodinamica-948974287-msmall.jpg");
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: cover;
+}
+
+.grid-item.a:hover {
+    background: gold;
+    /* background-image: url("https://mdn.mozillademos.org/files/6457/mdn_logo_only_color.png"); */
```
Comment (agandia9) [inline staff/PacoGD/portfolio/styles/main.css:74]: !!

File: staff/PacoGD/portfolio/styles/main.css [hunk 8]
```diff
@@ -0,0 +1,118 @@
...
+    max-height: 169px;
+    max-width: 320px;
+    position: absolute;
+    top: 0px;
+    right: 0px;
+    
+    
+}
+
+/* .grid-item.a:active {
```
[the remark this answers is not in the document]
Comment (PacoGD) [reply staff/PacoGD/portfolio/styles/main.css:94]: These are my notes!

File: staff/PacoGD/js/challenges/to-camel-case/scripts/toCamelCase.js [hunk 9]
```diff
@@ -0,0 +1,13 @@
+function toCamelCase(str){
```
Comment (agandia9) [inline staff/PacoGD/js/challenges/to-camel-case/scripts/toCamelCase.js]: use strict in all js files

**Pull request merged**